src/utils/StreamHighlightUtils.ts

```diff
@@ -77,13 +77,21 @@ export class StreamHighlightUtils {
 
 export function getRestHighlightsForAllTerms(
   toHighlight: string,
-  termsToHighlight: { [originalTerm: string]: string[] },
-  phrasesToHighlight: { [phrase: string]: { [originalTerm: string]: string[] } },
+  termsToHighlight: IHighlightTerm,
+  phrasesToHighlight: IHighlightPhrase,
   opts: IStreamHighlightOptions
 ): IHighlight[] {
   const indexes = [];
   const sortedTerms = _.keys(termsToHighlight).sort(termsSorting);
-  _.each(sortedTerms, (term: string) => {
+
+  const phrasesTerms = _.chain(phrasesToHighlight)
```

Comment:
What about just `phrases` for the variable name here?

Reply:
I think it's important to specify that its the terms inside the phrase, not the phrase itself

Reply:
Ok, fair point. I would just ask to adjust the spelling to `phraseTerms` please.